### DISCUSSION TITLE
Snapper and low disk space

### DISCUSSION THREAD

**Discussion by carlosEgarc** [SpiralLinux/SpiralLinux-project#517] [General]
Hola, muy buenos días. Cordial saludo. 

El día de ayer mi SSD (240GB) me estaba informando de bajo espacio disponible en disco. Revisando veo que el directorio de Snapshots esta usando unos 35 GB.  Por no preguntar antes, tome la decisión de des-instalar Snapper,  y borrar manualmente el directorio. Pero aun con el comando sudo -rm me dice que el directorio es de solo lectura y no me permite borrar el directorio.

Nuevamente instale Snapper pero ahora me dice que no tiene archivo de configuración, Hay alguna forma que pueda prescindir de las  instantáneas snapper ? La verdad es que solo tengo en ese equipo el juego Euro Truck Simulator y X-plane. Y borrando el archivo snapper el disco me quedaría con unos 70 Gb libres,

La "protección" de Snapper en este caso no me interesa tanto por que es un equipo que no se me perdería nada importante en caso de falla.

Nota final : Soy usuario de Linux ya hace unos 10 años pero ya la parte técnica no le trabajo tanto.  

De antemano gracias por tu ayuda

Carlos Garcia

**Comment by geckolinux** [comment]
Hola Carlos, en el caso que describes tal vez sería mejor volver a instalar SpiralLinux con otro sistema de archivos como XFS.
Pero si prefieres conservar tu instalación actual puedes usar el programa **Snapper GUI** que viene por defecto con SpiralLinux para borrar las instantáneas (nota que hay instantáneas del sistema **root** y otras de tus archivos personales **home**):
![image](https://github.com/user-attachments/assets/94d4d665-de9d-419a-ac78-cda0d3993f9e)

Y también puedes usar el mismo programa y seleccionar la opción **Propiedades** para cambiar o desactivar la configuración de instantáneas automáticas:
![image](https://github.com/user-attachments/assets/e5f265c4-f07d-452c-a839-721c539e824c)

Espero haberte ayudado. Saludos cordiales.

**Reply by carlosEgarc** [reply]
Hola, Gracias por tu amable respuesta. 

Si, entiendo tu respuesta, pero como des instale Snapper (pensando que se borraban las snapshots) Ahora que lo volví a instalar no me salen las opciones de Root y de Home. Pienso que hay alguna configuración que hace falta.

Creo que me faltaría "generar" un archivo de configuración. O si es posible una imagen donde dice propiedades para colocarlos igual y pienso que se restauraría la cosa.

Carlos Garcia

**Reply by geckolinux** [reply]
Hola de nuevo. En ese caso probablemente sería más rápido volverlo a instalar con XFS o EXT4.

**Comment by carlosEgarc** [comment]
Feliz y prospero 2025 estimado. 

A la final compre un disco SSD de 540 GB y "clone" el disco de 240 Gb. Intente con Clonezilla pero no me funciono, Termine usando Foxclone, https://foxclone.org/ Proceso super rápido y sencillo. Al final con Gparted ajuste el tamaño de la partición para que tomara todo el espacio del disco. 

Es decir sigo funcionando con Spiral Linux Cinnamon. al momento todo sigue funcionando al 100 % 

Gracias nuevamente por tu tiempo y amable respuesta

Carlos Garcia

**Reply by geckolinux** [reply]
Me alegro que hayas encontrado una buena solución de acuerdo a tus necesidades. Gracias por informarnos, un saludo caluroso.
